src/dune_sexp/syntax.mli

```diff
@@ -121,4 +121,6 @@ val key : t -> Key.t Univ_map.Key.t
 
 val get_exn : t -> (Version.t, 'k) Decoder.parser
 
+val get : t -> (Version.t option, 'k) Decoder.parser
```

Comment:
I added the option version of get_exn since I want to ignore the failure during the version checks. The failure is the fact that menhir lang is not declared, and when validating menhir flags its not relevantl